index.html

```diff
@@ -2,236 +2,338 @@
+              </p>
+              <ul class="stacks">
+                <li><a href="#">html</a></li>
+                <li><a href="#">bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <button class="btn btn-main">See Project</button>
+          </div>
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Website Portfolio</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="stacks">
+                <li><a href="#">html</a></li>
+                <li><a href="#">bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <button class="btn btn-main">See Project</button>
+          </div>
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Profesional Art Printing Data More</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="stacks">
+                <li><a href="#">html</a></li>
+                <li><a href="#">bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <button class="btn btn-main">See Project</button>
+          </div>
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Data Dashboard Healthcare</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="stacks">
+                <li><a href="#">html</a></li>
+                <li><a href="#">bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <button class="btn btn-main">See Project</button>
+          </div>
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Website Portfolio</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="stacks">
+                <li><a href="#">html</a></li>
+                <li><a href="#">bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <button class="btn btn-main">See Project</button>
+          </div>
+        </div>
+      </section>
+      <section class="about-me">
```

Comment:
- [ ] In **Desktop version**, The `background image`, and `lower border` is missing kindly add them.
- Your website 
![Capture](https://user-images.githubusercontent.com/26767383/154798354-aa6d126a-0655-420d-8e08-826e81b6c27a.PNG)

- Original website
![ergrg](https://user-images.githubusercontent.com/26767383/154798378-86faa26a-c291-4070-afa7-710e4c863164.PNG)

Reply:
Change made